### DISCUSSION TITLE
Benefits for moving from OSM as base map to OvertureMaps

### DISCUSSION THREAD

**Discussion by mpaine-act** [OvertureMaps/data#182] [Discussion]
I have an open issue @ [OpenTripPlanner repo](https://github.com/opentripplanner/OpenTripPlanner/issues/5269) regarding any advantages for changing base maps from OSM to OM.   Since OM expands on OSM, but not necessarily regarding street geography and wayfinding/transit related features, we don't yet know if it is worth the investment to consider this conversion.  

Perhaps your team can better answer this question for us.   Thank you.

**Comment by ViolaineDo** [comment]
#66 (similar issue)

**Comment by brad-richardson** [comment]
Hi @mpaine-act, apologies for the (very) delayed response here.

Transportation is in the process of expanding the road network to include ML-derived geometries from aerial imagery (non-OSM) directly into our delivery, which might be interesting to you. We think there's also value provided by the schematized and normalized data - things like turn and access restrictions in particular are much easier to ingest than OSM tags. If there are any features that would block you from adopting Overture, let us know! I'm converting this to a discussion for any additional follow-up.

**Reply by brad-richardson** [reply]
We now include non-OSM roads in Saudi Arabia and Italy to start and will continue to expand globally
